analysis/classification.py

```diff
@@ -432,10 +432,8 @@ def read_training_log_from_csv(self, directory):
             progression log
         """
         super().read_training_log_from_csv(directory)
-        self._train_log_accuracy = np.mean(self._log_train[:, :, 3], axis=0)
-        self._val_log_loss = self._log_val[:, 1]
```

Comment:
Does the removal of loss here mean we lose the ability to plot? Or is this removal of duplicate code somewhere else?

Reply:
it's still available, just moved to the base class [here](https://github.com/WatChMaL/WatChMaL/pull/31/files/2ad1c240f5d184e947b361416e9a950a33a5d89d#diff-11dd39b5e5e4b11e71305b89d13495e88c2b55d96c11ddad010e093ff92c14fcR136) so it can be used for both classification and regression.